messages/btc.options

```diff
@@ -14,12 +14,13 @@
 
 BTCScriptConfig.Multisig.xpubs max_count:15
 BTCPubRequest.keypath max_count:10
-BTCSignInitRequest.keypath_account max_count:10
+BTCSignInitRequest.script_configs max_count:3
```

Comment:
Why only 3? Is it basically to allow a max of one `P2WPKH_P2SH`, `P2WPKH` and `Multisig` each? What about the other coins? I guess I don't have a clear picture of how the unified accounts will look like. Are they going to be grouped by the type, like BTC, ETH, etc?

Reply:
Yeah, we currently only have really 2 types that can be used at once (segwit and native segwit), and in the future possibly also taproot, so 2 would actually be enough.

multisig is treated separately.

the number here is kind of arbitrary, it could also be 10. it's just so nanopb knows how much memory to allocate for the array.

Reply:
when you start a sign workflow, you specify the btc-like coin: currently BTC or LTC.

ETH is completely separate and does not need unified accounts (there is only one eth account now, while there are two to three btc accounts).

Reply:
Oh, right. Somehow I was thinking about inputs, not accounts. And multisig is in `BTCScriptConfig_multisig_tag`.
Yeah, probably good to keep this max count low.